### PR TITLE
Stellarium 24.2

### PR DESCRIPTION
**Pull request by alex-w**
Switch to new version of Stellarium

### CODE REVIEW

**Comment by Begasus** [comment]
I presume the checksum would have changed atleast?

File: sci-astronomy/stellarium/stellarium-24.2.recipe
Comment (jmairboeck) [inline sci-astronomy/stellarium/stellarium-24.2.recipe:1]: This will need at least a change of CHECKSUM_SHA256.

**Comment by Begasus** [comment]
Thanks, buildmasters are stuck at the moment, so we probably should wait for someone to push a button there.

Me pokes @mmlr (could you have a look?)

**Comment by Begasus** [comment]
Build error on 64bit (32bit still at it): https://build.haiku-os.org/buildmaster/master/x86_64/logviewer.html?buildruns/10949/builds/82612.log

**Pull request merged**